Classes/Controller/Backend/NotificationApplicationController.php

```diff
@@ -0,0 +1,154 @@
+<?php
+namespace PAGEmachine\Ats\Controller\Backend;
+
+use PAGEmachine\Ats\Application\ApplicationFilter;
+use PAGEmachine\Ats\Message\RejectMessage;
+use PAGEmachine\Ats\Message\ReplyMessage;
+use PAGEmachine\Ats\Service\PdfService;
+
+/*
+ * This file is part of the PAGEmachine ATS project.
+ */
+
+
+/**
+ * NotificationApplicationController
+ * Used in the "Mass Notification" Module
+ */
+class NotificationApplicationController extends ApplicationController
+{
+    /**
+     * Action URLs for the action menu
+     *
+     * @var array
+     */
+    protected $menuUrls = [];
+
+    /**
+     * Forwards to the first allowed action (since some could be disallowed by role)
+     *
+     * @return void
+     */
+    public function initializeIndexAction()
+    {
+
+        $this->forward("listAll");
+    }
+
+    /**
+     * Lists applications for the mass notification in the backend module
+     *
+     * @param  ApplicationFilter $filter
+     * @param  bool $resetFilter
+     * @param  RejectMessage $rejectMessage
+     * @param  ReplyMessage $ReplyMessage
+     * @param  string $messageType
```

Comment:
I think this should use an `Enumeration`

Reply:
@mbrodala this needs to be refactored in a few places, so it's fine for this PR to use the constant string.